packages/website/gatsby-node.js

```diff
@@ -74,3 +74,11 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
     });
   }
 };
+
+exports.onCreateWebpackConfig = ({ actions }) => {
```

Comment:
I don't know what's up with this. We had to add the same to our config for `react-view` to work. 

re: https://github.com/uber/react-view/pull/18#issuecomment-573157256